src/docs/sdk/metrics.mdx

```diff
@@ -43,9 +43,8 @@ exist:
 * **Background Aggregation:** SDKs are encouraged to defer flushing and aggregation
   into a background thread.  For SDKs where `fork()` is to be expected, the background
   thread needs to ensure it restarts after fork (eg: python).
-* **Tagging:** metrics are tagged by key value pairs, where each key can have more
-  than one value.  Both keys and values are limited to strings, but the SDK can
-  expose other types if it has a reasonable way to stringify these.
+* **Tagging:** metrics are tagged by key value pairs. Both keys and values are limited to strings,
```

Comment:
```suggestion
* **Tagging:** metrics can be tagged with a unique key and an optional value for each key. Both keys and values are limited to strings,
```

Maybe something like this, since there can be *no* tags at all and tags *without* values.